pkg/iss/iss.go

```diff
@@ -144,6 +151,10 @@ func New(
 
 	// TODO: Make sure that startingChkp is consistent with params.
 
+	leaderPolicy, err := issutil.LeaderPolicyFromBytes(startingChkp.Snapshot.EpochData.LeaderPolicy)
+	if err != nil {
+		logger.Log(logging.LevelWarn, "Error deserializing leader selection policy from checkpoint", err)
+	}
```

Comment:
I think here the error should be propagated upwards (i.e. returned), and the log message can have an error level. Without a leader selection policy, the system cannot function.

Reply:
Done.